File: level1/수박수박수박수박수박수/solution.js
function solution(n) {
  var subak = ["수", "박"];
  var answer = "";
  for (var i = 0; i < n; i++) {
    answer += subak[i % 2];
  }
  return answer;
}

console.log(solution(3));
console.log(solution(4));

File: level1/수박수박수박수박수박수/Solution.java
class Solution {
    public String solution(int n) {
        char[] subak = {'수', '박'};
        StringBuilder answer = new StringBuilder();
        for (int i = 0; i < n; i++) {
            answer.append(subak[i % 2]);
        }
        return answer.toString();
    }

    public static void main(String[] args) {
        Solution s = new Solution();
        System.out.println(s.solution(3));
        System.out.println(s.solution(4));
    }
}
